aws_topology/stackstate_checks/aws_topology/resources/registry.py

```diff
@@ -0,0 +1,63 @@
+from six import with_metaclass
+from ..utils import correct_tags, capitalize_keys
+
+
+class ResourceRegistry(type):
+
+    REGISTRY = {}
+
+    def __new__(cls, name, bases, attrs):
+        new_cls = type.__new__(cls, name, bases, attrs)
+        """
+            Here the name of the class is used as key but it could be any class
+            parameter.
+        """
+        if new_cls.API != '??':
+            if (cls.REGISTRY.get(new_cls.API) is None):
```

Comment:
Remove redundant parentheses

Reply:
Haha good catch. Too long nosing around in nodejs makes you do this :-)